05-cond.md

```diff
@@ -65,18 +65,17 @@ which is short for "else if".
 This makes it simple to write a function that returns the sign of a number:
```

Comment:
This line should be reworded to not reference functions. Perhaps something like "The following Python code uses `elif` to print the sign of a number."